### ISSUE TITLE
There is no CollisionMesh in examples

### ISSUE THREAD

**Issue by CrossroadInTheVoid** [3dreamengine/3DreamEngine#63]
There is no CollisionMesh in examples and it works totally unclean.

It worked like this in the past:

```
collision = require("3DreamEngine/collision");
...
self.box_collider_mesh = dream:loadObject("models/collider_box", {splitMaterials = false, noCleanup = true});
...
local _collider = collision:newMesh(self.box_collider_mesh);
...
_collider:moveTo (vec3(_vector_a.x, 0, -1 * _vector_a.y)); --or any other action with created collider
```

But as I see it was changed. What should be included to fix

`attempt to index global 'CollisionMesh' (a nil value)`

error? What parametres should I use?
`CollisionMesh:newCollisionMesh(mesh, shapeMode)`
What the mesh is? Loaded mesh object? Or a string path to an obj file? Link at documentation looks dead.

P. S. I'm thisnickwasfree, I've lost access to that acc so I've created new one.

**Comment by Luke100000** [comment]
Welcome back!

The physics has been extracted into its own extension: https://3dreamengine.github.io/3DreamEngine/docu/extensions/physics

Specifically this is how to create a shape (the prepared collision mesh) and `world:add` (which then creates an collider)
```lua
--create a shape from this object and it's current transformation
local shape = physics:newObject(object)

--add this shape to the world
local collider = world:add(shape, "dynamic", x, y, z)
```

**Comment by CrossroadInTheVoid** [comment]
Thanks. But:
`Error: extensions/physics/world.lua:261: attempt to index field 'physics' (a nil value)`
Where is the creation of love.physics table?

**Comment by Luke100000** [comment]
Either in the conf.lua https://love2d.org/wiki/Config_Files or you can require it manually `love.physics = require("love.physics")`

**Comment by CrossroadInTheVoid** [comment]
Oh, it was switched off in config. Thank you. And sorry for disturbing, take my questions as «dumb user test» for your project.

**Comment by CrossroadInTheVoid** [comment]
And some more questions.
Method moveTo removed. So, how to move collider to vec3 coordinates? I guess applyForce or applyLinearImpulse are not the best way.
I've used colliders to detect mouseclicks at mobs and raycasts between mobs. Looks like it's not good choice for such a thing now and I should use RaytraceMesh?

**Comment by Luke100000** [comment]
Yes, use raytracing for that. The physics extension is 2.5D and is based on Box2d. Setting position is an oversight from me, will add that, but it boils down to `collider:getBody():setPosition(x, z)` and `collider.y = y` where body is a love2d physics body.
